### DISCUSSION TITLE
Run function after template is inserted

### DISCUSSION THREAD

**Discussion by samandar-boymurodov** [SilentVoid13/Templater#1250] [Help]
Is there any way to listen for template insertion into a file and execute a certain function against that file?
There is a "Startup Templates" settings. I linked a note that contains example function in https://silentvoid13.github.io/Templater/internal-functions/internal-modules/hooks-module.html?highlight=hook#examples. But that function is never run when I use other templates.

**Comment by Zachatoo** [comment]
What's the use case?
